src/app/(with-navbar)/components/[type]/page.tsx

```diff
@@ -57,6 +57,7 @@ export default async function ComponentPage({
 
   const mdxDoc = await getInstallationDoc({
     componentType: type,
+    // @ts-expect-error async component
```

Comment:
Tbh, I'm not sure if adding `@ts-expect-error` is something we'd want in our project, but I feel like I can't really make that decision on my own.

Reply:
Yep, I am not sure either. Just to expand I would be ok with that if I feel that its very likely this feature gonna be supported soon. But I am not really confident about that
Also thats way I extracted this into separate proposal. Maybe we will switch to @next/mdx and this wont be relevant anymore ...

Reply:
I'm looking into `@next/mdx` and we will see if that is a good option for us. I'm still not sure if it would be good for us.